### ISSUE TITLE
Missing `subscribeKeys(store, keys, cb)` method ?

### ISSUE THREAD

**Issue by Jeremboo** [nanostores/nanostores#262]
We have both `store.subscribe(cb)` and `store.listen(cb)` to decide if we want to call the listeners immediately on subscribe or not.

So why not having the same with `subscribeKeys(store, keys, cb)` along `listenKeys(store, keys, cb)` ?

```typescript
import { listenKeys, type StoreValue } from 'nanostores';

export function subscribeKeys<
  SomeStore extends { get: () => StoreValue<SomeStore>; setKey: (key: any, value: any) => void }
>(
  $store: SomeStore,
  propsToListen: SomeStore extends { setKey: (key: infer Key, value: never) => unknown } ? readonly Key[] : never,
  callback: (_props: StoreValue<SomeStore>) => void
) {
  listenKeys($store, propsToListen, callback);
  callback($store.get());
}
```

I'm using it in prod and it's working pretty good !

**Comment by ai** [comment]
Do you have a real use case for that? If yes, we can add.

**Comment by Jeremboo** [comment]
The use case is similar to using the already implemented `store.subscribe(cb)` method.
It's to trigger the callback immediately on subscribe.

I'm using it in vanillaJS. For instance to init my JS classes with values from the store.

Exemple :

```typescript
export default class MyExample {
  valueA = 0;

  constructor() {
    subscribeKeys(webgl, ['valueA'], this.handlePropsChanged);
  }

  handlePropsChanged = ({ valueA }) => {
    if (valueA != this.valueA) {
      // Animate with the new values directly instantiation and all the other times
      gsap.to(this, { valueA, valueB, duration: 1 });
    }
  }
}
```

If it's a use case you don't want to support, maybe for consistency `store.subscribe(cb)` should be removed ?

**Comment by ai** [comment]
Send PR with a function to `next` branch

**Comment by gismya** [comment]
This was released with [0.11.0](https://github.com/nanostores/nanostores/releases/tag/0.11.0)
